src/components/Buttons/UpVote.jsx

```diff
@@ -5,8 +5,8 @@ const UpVote = ({
 	allReviews,
 	setAllReviews,
 	setReview,
-    setLikes,
-    likes
+	setLikes,
+	likes,
 }) => {
 	const Up = () => {
```

Comment:
proper camelCase needed here (and maybe a bit more descriptive than "up" ... up what? 🤔)